tests/tasks/el_repo_setup.yml

```diff
@@ -1,4 +1,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
+- name: Gather the minimum subset of ansible_facts required
+  setup:
+    gather_subset: min
```

Comment:
please add a `when` condition like this https://github.com/linux-system-roles/network/blob/main/tasks/set_facts.yml#L6
gathering facts is an expensive operation, even for `gather_subset: min` - if the playbook already gathered facts, you do not want to gather them again